client/src/components/InventoryTable.js

```diff
@@ -1,37 +1,45 @@
-import React from 'react'
+import React from 'react';
+
+var width = {
+  width: '100%',
+  textAlign: 'center',
+};
+
+const renderTable = items => {
+  console.log(items)
+  return items ?  (
+    <table style={width}>
+      <tr>
+        <th scope="col">#</th>
+        <th scope="col">Available</th>
+        <th scope="col">Name</th>
+        <th scope="col">Description</th>
+        <th scope="col">Owner</th>
+      </tr>
+      {items.map(item => (
+        <tr>
+          <th scope="row">{item.numberAvailable}</th>
```

Comment:
This column does not represent number of items. It is an accessibility feature for screen readers. This should just be a sequential count for every row rendered (starting at 1). I think you should incorporate the number available into the 'Available' column. Also can you rename `numberAvailable` property to `quantity`?